docs/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+# Issue Template
+
+* Generally speaking, issues should consist of bugs in SLEAP that will require a pull request. For general questions and suggestions for enhancements, you can start a new discussion [here](https://github.com/talmolab/sleap/discussions)
+
+## Description of issue
+
+* Please provide a clear and concise explanation of the issue
+
+## Expected behavior
+
+* Please provide a clear and concise explanation of the behavior you are expecting
+
+## Versions:
+- OS + version [e.g Linux Ubuntu 18.04]
+- Browser + version [e.g Firefox 100.0]
+- SLEAP version [e.g v1.2.3]
```

Comment:
We could try adding check boxes for some of these fields to make it easier for users (and easier to extract the information we want), but it could get difficult if we get too specific (always include an "Other" option). What do you think?